rudaux/flows.py

```diff
@@ -177,8 +192,9 @@ def build_autoext_flows(config, args):
                 # Fill in submission deadlines
                 submission_sets = subm.build_submission_set.map(unmapped(config), submission_sets)
 
-                # Compute override updates
-                overrides = subm.get_latereg_overrides.map(unmapped(config.latereg_extension_days[group]), submission_sets)
+                # Compute override updates if registration deadline has not passed.
+                if plm.now().in_timezone(config.notify_timezone).timestamp() < plm.from_format(config.registration_deadline, 'YYYY-MM-DD').timestamp():
```

Comment:
@trevorcampbell 

Just thinking about this.

We might need to register whether rudaux ran the `get_latereg_overrides` at least once after the deadline, otherwise, we will miss students that registered close to the deadline. 

Right?